<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>progettoQt</class>
 <widget class="QWidget" name="progettoQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProgettoQt</string>
  </property>
  <widget class="QLineEdit" name="urlBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>255</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>url</string>
   </property>
  </widget>
  <widget class="QPushButton" name="analyzeButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Analizza</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="wordLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>501</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="charLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="noSpaceLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="phraseLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>130</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="paraLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
